client/src/app/shared/shared-main/misc/channels-setup-message.component.ts

```diff
@@ -16,14 +16,11 @@ export class ChannelsSetupMessageComponent implements OnInit {
     private authService: AuthService
   ) {}
 
-  get userInformationLoaded () {
-    return this.authService.userInformationLoaded
-  }
-
   get hasChannelNotConfigured () {
-    return this.user.videoChannels
-      .filter((channel: VideoChannel) => (!channel.avatar || !channel.description))
-      .length > 0
+    if (this.user.videoChannels)
```

Comment:
Prefer `if (!this.user.videoChannels) return` form

Reply:
it's weird with this form the alter does not show anymore

Reply:
Thanks for the tip :+1: